### ISSUE TITLE
there is no InteractivelyCreateDestroyGraph 

### ISSUE THREAD

**Issue by farzadhallaji** [paulbrodersen/netgraph#13]
```
Traceback (most recent call last):
  File "test.py", line 12, in <module>
    plot_instance = netgraph.InteractivelyCreateDestroyGraph(graph, draw_arrows=True, ax=ax)
AttributeError: module 'netgraph' has no attribute 'InteractivelyCreateDestroyGraph'

```

**Comment by farzadhallaji** [comment]
I got it!
just use InteractivelyConstructDestroyGraph instead of InteractivelyCreateDestroyGraph

**Comment by paulbrodersen** [comment]
Great catch. No idea why I renamed that. 
That part of the code is still very much in its infancy. If you have any suggestions (or pull requests), I am all ears!

Issue now fixed in b7efe8f (corrected name in documentation).
